core/src/main/resources/error/error-classes.json

```
@@ -1730,6 +1730,11 @@
           "Pandas user defined aggregate function in the PIVOT clause."
         ]
       },
+      "PARAMETERIZED_COMMAND" : {
+        "message" : [
+          "Query parameters in Command."
```

Is "query parameters" a term? I thought we go by "parameter marker". Either way we can be more descriptive.

```suggestion
          "Parameter markers in a DDL statement. Parameter markers must only be used in a query, or DML statement. Usage within CREATE or ALTER is unsupported."
```